bundles/org.openhab.binding.plex/README.md

```diff
@@ -0,0 +1,172 @@
+
+`.items` file
+
+```
+String    BridgePlexCurrent            "Current players"           {channel="plex:server:plexrServer:currentPlayers"}
+String    BridgePlexCurrentActive      "Current players active"    {channel="plex:server:plexrServer:currentPlayersActive"}
+Switch    PlexTVPower01                "Power"                     {channel="plex:player:MyViewerName01:power"}
+String    PlexTVStatus01               "Status [%s]"               {channel="plex:player:MyViewerName01:state"}
+String    PlexTVTitle01                "Title [%s]"                {channel="plex:player:MyViewerName01:title"}
+String    PlexTVType01                 "Type [%s]"                 {channel="plex:player:MyViewerName01:type"}
+String    PlexTVEndTime01              "End time"                  {channel="plex:player:MyViewerName01:endtime"}
+Dimmer    PlexTVProgress01             "Progress [%.1f%%]"         {channel="plex:player:MyViewerName01:progress"}
+String    PlexTVCover1                 "Cover"                     {channel="plex:player:MyViewerName01:thumb"}
+String     ShellArt01                  "Background art"            {channel="plex:player:MyViewerName01:art"}
+Switch    PlexTVPower02                "Power"                     {channel="plex:player:MyViewerName02:power"}
+String    PlexTVStatus02               "Status [%s]"               {channel="plex:player:MyViewerName02:state"}
+String    PlexTVTitle02                "Title [%s]"                {channel="plex:player:MyViewerName02:title"}
+String    PlexTVType02                 "Type [%s]"                 {channel="plex:player:MyViewerName02:type"}
+String    PlexTVEndTime02              "End time"                  {channel="plex:player:MyViewerName02:endtime"}
+Dimmer    PlexTVProgress02             "Progress [%.1f%%]"         {channel="plex:player:MyViewerName02:progress"}
+String    PlexTVCover2                 "Cover"                     {channel="plex:player:MyViewerName02:thumb"}
+String    ShellArt02                   "Background art"            {channel="plex:player:MyViewerName02:art"}
+```
+
+`.rules` file
+
+```
+rule "Send telegram with title for My Viewer Name 01"
+when
+    Item PlexTVTitle01 changed
+then
+    val telegramActionPlexBot = getActions("telegram","telegram:telegramBot:PlexBot")
+    telegramActionPlexBot.sendTelegram("Bedroom Roku is watching %s", PlexTVTitle01.state.toString)
+end
+
+rule "Send telegram with title for My Viewer Name 02"
+when
+    Item PlexTVTitle02 changed
+then
+    val telegramActionPlexBot = getActions("telegram","telegram:telegramBot:PlexBot")
+    telegramActionPlexBot.sendTelegram("Bedroom Roku is watching %s", PlexTVTitle02.state.toString)
+end
+```
+
+You can also design some rules from the openHAB UI:
+
+![Dim lights when playing movie](doc/Rules_DimLightsWhenPlayingMovie.png)
+![Adjust lights when movie is paused](doc/Rule_AdjustLightWhenMovieIsPaused.png)
+![Enable lights when movie is stopped](doc/Rule_EnableLightsWhenPlayerIsStopped.png)
```

Comment:
The readme should contain any specific UI details. You could post example use cases in the samples section of the community forum. See https://community.openhab.org/c/tutorials-examples/10